src/runtime-rs/crates/hypervisor/src/lib.rs

```diff
@@ -45,14 +46,7 @@ const VM_ROOTFS_FILESYSTEM_EXT4: &str = "ext4";
 const VM_ROOTFS_FILESYSTEM_XFS: &str = "xfs";
 const VM_ROOTFS_FILESYSTEM_EROFS: &str = "erofs";
 
-// before using hugepages for VM, we need to mount hugetlbfs
-// /dev/hugepages will be the mount point
-// mkdir -p /dev/hugepages
-// mount -t hugetlbfs none /dev/hugepages
-const DEV_HUGEPAGES: &str = "/dev/hugepages";
 pub const HUGETLBFS: &str = "hugetlbfs";
-const SHMEM: &str = "shmem";
-const HUGE_SHMEM: &str = "hugeshmem";
```

Comment:
I want to confirm that the change is for compilation?

Reply:
Yes, they are only being used by `Dragonball`. So I faced with the `never used` error unless I move them to the hypervisor using them.